internal/connector/internal/api/dbapi/processor.go

```diff
@@ -0,0 +1,143 @@
+package dbapi
+
+import (
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/api"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/db"
+)
+
+type ProcessorDesiredState string
+type ProcessorStatusPhase string
+
+const (
```

Comment:
I'm assuming the same life-cycle is to be used for `Processor`'s.

Reply:
I think so yes.

Reply:
Me too. Are you happy me keeping the separate enums, constants, transition definitions etc (I.e. duplicating connectors) or shall we consider reconciling into a single shared approach? It could be a new JIRA and new future PR.